package.json

```diff
@@ -13,49 +13,55 @@
     "dbLocation": "mongodb://localhost/didactic-disco"
   },
   "scripts": {
-    "start": "NODE_ENV=development parallelshell 'npm run express-dev-server' 'npm run webpack-dev-server'",
+    "lint": "eslint --ext .jsx app",
+    "lint:fix": "npm run lint -- --fix",
+    "pretest": "npm run lint",
+    "test": "echo 'no tests available' && exit 1",
+    "bundle": "NODE_ENV=production webpack --config webpack/config.dist.js --progress",
```

Comment:
adding in some new commands here. these can be played around with as you see fit.